### PR TITLE
Add lc-task-mc to the workflow

### PR DESCRIPTION
**Pull request by DelloStritto**

### CODE REVIEW

**Comment by vkucera** [comment]
Is this PR ready for review?

**Comment by DelloStritto** [comment]
Hi @vkucera . Yes, The PR is ready. I pushed also the updated "Compare.C" with the Lc plots in.

**Review by ginnocen — APPROVED** [review]
hi @DelloStritto @nzardosh this is good to go from my side, as soon as the O2 PR is merged https://github.com/AliceO2Group/AliceO2/pull/5013. Cheers GM

File: codeHF/dpl-config_run3.json [hunk 1]
```diff
@@ -113,8 +115,10 @@
     "hf-task-dplus": {
         "d_selectionFlagDPlus": "0"
     },
-     "hf-task-lc": {
-        "d_selectionFlagLc": "0"
+    "hf-task-lc": {
+        "cutEtaCandMax": "-1.",
+        "d_selectionFlagLc": "0",
+        "d_selectionFlagLcbar": "0"
```
Comment (vkucera) [inline codeHF/dpl-config_run3.json:121]: Configuration of `hf-task-lc-mc` is missing completely.

**Review by vkucera — CHANGES_REQUESTED** [review]

**Comment by vkucera** [comment]
@ginnocen We need to decide here whether the Lc task should be included in the list of tasks activated by default, on top D0 and D+.

File: codeHF/dpl-config_run3.json (hunk 1 above)
Comment (DelloStritto) [reply codeHF/dpl-config_run3.json]: Thanks @vkucera . I pushed it.

**Review by vkucera — CHANGES_REQUESTED** [review]
- [x] Please update also the Run 5 JSON.
- [x] Either Set `DOO2_TASK_LC=1` or remove the condition in `MakeScriptPostprocess`. Otherwise the `Compare.C` macro is not executed by default.
- [x] These histograms do not have matching binnings. Please synchronise with Francesco.
   - [x] `hInvMassLc`, `hf-task-lc/hmass`
   - [ ] `hDecLenLc`, `hf-task-lc/hdeclength`

File: codeHF/Compare.C [hunk 2]
```diff
@@ -99,9 +113,17 @@ Int_t Compare(TString filerun3 = "AnalysisResults.root", TString filerun1 = "Ver
...
                            "secondary vtx x - 3prong",
                            "secondary vtx y - 3prong",
-                           "secondary vtx z - 3prong"
+                           "secondary vtx z - 3prong",
+                           "3-prong mass (p K #pi)",
+                           "#Lambda_{c} pt cand",
+                           "#Lambda_{c} pt prong 0",
+                           "#Lambda_{c} pt prong 1",
+                           "#Lambda_{c} pt prong 2",
+                           "#Lambda_{c} Decay Lenght",
```
Comment (vkucera) [inline codeHF/Compare.C:122]: Fix typo

**Review by vkucera — CHANGES_REQUESTED** [review]

**Comment by vkucera** [comment]
To merge after https://github.com/AliceO2Group/AliceO2/pull/5150

**Comment by ginnocen** [comment]
hi @DelloStritto @vkucera can we resolve these conflicts and check that everything works so that we can merge it?

**Comment by DelloStritto** [comment]
@ginnocen @vkucera I has some issues with git so I closed this pull request and I opened a new one with no conflicts.

**Comment by ginnocen** [comment]
@DelloStritto, no pb, thanks!

**Pull request closed without merging**